### DISCUSSION TITLE
DNS Certificate Issue

### DISCUSSION THREAD

**Discussion by arnau01** [vercel/community#3405] [Help] [answered]
### Summary

Hi, I quickly changed to Firebase hosting but want to revert to Vercel. Have changed all the DNS records back but I'm now getting a certificate issue (i think it's still getting the certificate from Firebase) making me site inaccesible. Any suggestions on how to make the DNS point to my valid Vercel certificate. 

Thank you :)
<img width="962" alt="Screenshot 2023-07-27 at 22 40 55" src="https://github.com/vercel/community/assets/62598895/a8355572-d0b8-403c-ac80-89df7ac1ae6f">
<img width="975" alt="Screenshot 2023-07-27 at 22 40 07" src="https://github.com/vercel/community/assets/62598895/ea52d360-53d0-4501-baef-a63027e35fad">

### Example

throxy.com

### Steps to Reproduce

Changed DNS records to a different provider (Firebase Hosting)

Then reverted back to records from Vercel

Now showing certificate not valid.

Everything seems fine on Vercel side

**Comment by amyegan** [comment, marked as answer]
Hey @arnau01. It's normal for record propagation to take some time. It looks like your site is working correctly now. 🎉
